.circleci/config.yml

```diff
@@ -37,37 +37,6 @@ jobs:
         path: ~/project/out
         destination: crc
 
-  build_mock_release:
-    machine:
```

Comment:
The 10GB+ memory needed to build with the generated files could be mentioned in this commit log as a data point :)

Reply:
With the changes to use a string for the embedded data rather than a byte array, it should be possible to drop ` Remove mock release check for travis and circle CI ` ?